core/arch/riscv/plat-virt/platform_config.h

```diff
@@ -92,4 +93,8 @@
 #define RISCV_MTIME_RATE 1000000
 #endif
 
+#define PLAT_THREAD_EXCP_FOREIGN_INTR	\
```

Comment:
We have only one riscv platform, but that one can't use the default values. Does that make any sense?

Reply:
Actually, we expect that each riscv platform (even new platforms with new ISA extensions in the future) may define their own value.
So I proposed this solution.
Do you have any recommendation ?

Reply:
In that case, there's no need for a default value.

Reply:
> In that case, there's no need for a default value.

You mean we should define `THREAD_EXCP_FOREIGN_INTR` and `THREAD_EXCP_NATIVE_INTR` in each `platform_config.h`.
And remove the default value ?

Reply:
I think it's good to have a `PLAT_` prefix, but there's no need for the
``` C
#else
#define THREAD_EXCP_NATIVE_INTR		(CSR_XIE_SIE | CSR_XIE_TIE)
#endif
````
So in `core/arch/riscv/include/kernel/thread_arch.h` it would become:
``` C
#define THREAD_EXCP_FOREIGN_INTR	PLAT_THREAD_EXCP_FOREIGN_INTR
#define THREAD_EXCP_NATIVE_INTR		PLAT_THREAD_EXCP_NATIVE_INTR
```

Reply:
I've applied what @jenswi-linaro suggested.
@maroueneboubakri Do you have any comments ?